config/locales/en.yml

```diff
@@ -1792,9 +1792,9 @@ en:
         required_to_save_changes: (required to save changes)
         leave_blank_if_no_change: "leave blank if there's no change"
         submit_button_label: *update
-        delete_my_account: Delete my account
+        delete_my_account: Delete your account
         unregister: Unregister
-        confirm_are_you_sure: Are you sure your want to DELETE your account?
+        confirm_are_you_sure: Are you sure you want to DELETE your account?
```

Comment:
@achock You'll also have to change the corresponding lines in `sv.yml`.  That's the file that translates the I18N statements to Swedish.

You can use Google translates to convert to Swedish.  Then @thesuss can review and correct the language as needed.

And, it would be good to change the key from `delete_my_account` to `delete_your_account`.

Reply:
@achock You'll have to change references to that key in the code (including tests, if any).